src/murmur/main.cpp

```diff
@@ -292,11 +293,29 @@ int main(int argc, char **argv) {
 			detach = false;
 			qFatal("%s -- %s", qPrintable(args.at(0)), MUMBLE_RELEASE);
 		} else if (args.at(i) == QLatin1String("-license") || args.at(i) == QLatin1String("--license")) {
+#ifdef Q_OS_WIN
+			AboutDialog ad(NULL, AboutDialogOptionsShowLicense);
+			ad.exec();
+			return 0;
```

Comment:
On Windows, Murmur calls ShowMessageDialog() for qFatal. It's defined at https://github.com/mumble-voip/mumble/blob/54eab8d4bc2f5aa7ddd6b599202f9fe017011328/src/murmur/main.cpp#L115-L128

This is because Murmur on Windows is built a a GUI app with a GUI log, a systray icon, etc. -- we can't easily log to stdout/stderr, since there's no console.

So, I'm not really doing this just because it's nicer to have a gui on Windows. I'm doing it because Murmur on Windows is a GUI app, and implementing -license, -authors and -third-party-licenses using qFatal() causes a message dialog that's too large to fit on anyone's screen to show up.

I felt the best solution was to add an About dialog. Users running Murmur on Windows can now see version/license information while running it, and the CLI flags are now usable (you can scroll through the license texts).